pureconfig/src/main/scala/com/avast/clients/rabbitmq/pureconfig/PureconfigImplicits.scala

```diff
@@ -0,0 +1,94 @@
+package com.avast.clients.rabbitmq.pureconfig
+
+import _root_.pureconfig.ConfigReader.Result
+import _root_.pureconfig._
+import _root_.pureconfig.generic.ProductHint
+import _root_.pureconfig.generic.semiauto._
+import com.avast.clients.rabbitmq.api.DeliveryResult
+import com.avast.clients.rabbitmq.api.DeliveryResult._
+import com.avast.clients.rabbitmq.{
+  AutoBindExchangeConfig,
+  AutoBindQueueConfig,
+  AutoDeclareExchangeConfig,
+  AutoDeclareQueueConfig,
+  BindArgumentsConfig,
+  BindExchangeConfig,
+  BindQueueConfig,
+  ConsumerConfig,
+  CredentialsConfig,
+  DeclareArgumentsConfig,
+  DeclareExchangeConfig,
+  DeclareQueueConfig,
+  ExchangeType,
+  NetworkRecoveryConfig,
+  ProducerConfig,
+  ProducerPropertiesConfig,
+  PullConsumerConfig,
+  RabbitMQConnectionConfig,
+  RecoveryDelayHandlers,
+  SslConfig,
+  TrustStoreConfig
+}
+import com.rabbitmq.client.RecoveryDelayHandler
+import org.slf4j.event.Level
+
+private[pureconfig] class PureconfigImplicits(implicit namingConvention: NamingConvention = CamelCase) {
```

Comment:
> `CamelCase`

why don't we just use kebab case, which is PureConfig default?

Reply:
`CamelCase` is chosen for backwards compatibility because that's what we've been using in the past.